src/ConnectedMode/Suppressions/TimedUpdateHandler.cs

```diff
@@ -39,33 +40,56 @@ internal sealed class TimedUpdateHandler : IDisposable
         private readonly IServerHotspotStoreUpdater serverHotspotStoreUpdater;
         private readonly IQualityProfileUpdater qualityProfileUpdater;
         private readonly ILogger logger;
+        private readonly IActiveSolutionBoundTracker activeSolutionBoundTracker;
 
         private bool disposed;
 
         [ImportingConstructor]
         public TimedUpdateHandler(ISuppressionIssueStoreUpdater suppressionIssueStoreUpdater,
             IServerHotspotStoreUpdater serverHotspotStoreUpdater,
             IQualityProfileUpdater qualityProfileUpdater,
-            ILogger logger)
-            : this(suppressionIssueStoreUpdater, serverHotspotStoreUpdater, qualityProfileUpdater, logger, new TimerFactory()) { }
+            ILogger logger,
```

Comment:
Minor: the convention is to put `ILogger` last in the non-optional parameter list (the reasoning being that it is a boilerplate/utility class, and the other imports are more important from the point of view of understanding what the importing class so they should come first).

Reply:
`ILogger` should be the last argument in this constructor too.